kratos/containers/system_vector.h

```diff
@@ -88,6 +88,13 @@ class SystemVector final
         mData.resize(size,false);
     }
 
+    SystemVector(const Vector& data, DataCommunicator& rComm=ParallelEnvironment::GetDataCommunicator("Serial")){
```

Comment:
```suggestion
    SystemVector(
        const Vector& data,
        DataCommunicator& rComm = ParallelEnvironment::GetDataCommunicator("Serial")) {
```
Minor style guide suggestion.